lib/uniris/oracles/supervisor.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Uniris.Oracles.Supervisor do
+  @moduledoc false
+
+  use Supervisor
+
+  alias Uniris.Oracles.{
+    Coingecko,
+    OracleCronServer
+  }
+
+  alias Uniris.Utils
+
+  def start_link(args) do
+    Supervisor.start_link(__MODULE__, args, name: __MODULE__)
+  end
+
+  def init(_args) do
+    optional_children = [
+      {OracleCronServer, [mfa: {Coingecko, :fetch, [DateTime.utc_now()]}, interval: 5_000],
+       id: :a},
+      {OracleCronServer,
```

Comment:
I don't get the use of this second oracle scheduler and moreover with this particular date

Reply:
nevermind